### ISSUE TITLE
popsize

### ISSUE THREAD

**Issue by pharringtonp19** [RobertTLange/evosax#1]
Congrats on a great library that is super easy to jump into!

To better understand how things work, I am running `Differential_ES` on regression problem ([Colab Notebook](https://gist.github.com/pharringtonp19/80694ad2a77da1222fc85ff03d00ce91))

One thing that I'm running into in this toy example is that the performance becomes worse when I increase the `popsize` of the `Differential_ES` strategy. Have you ever encountered something like this?

Attached are two figures corresponding to a population training size of [10.pdf](https://github.com/RobertTLange/evosax/files/8192751/10.pdf) and 
[500.pdf](https://github.com/RobertTLange/evosax/files/8192752/500.pdf)

**Comment by RobertTLange** [comment]
Hi @pharringtonp19 😄 

Thank you again for actively probing `evosax` and the kind words! In general stable ES hyperparameters can depend on the population size (similar to how the GD learning rate can depend on the batchsize). If in your example one changes the differential weight (`diff_w`) from the default of 0.8 to 0.5 you get much more stable dynamics for a population size of 500 (see attached image). For a population size of 10, on the other hand, this will slow down the progress/dampen the exploration of the Differential ES.

I am actively working on setting good default parameters for all strategies which are robust across standard tasks (supervised learning, control, etc.). But tuning parameters across tasks and strategies takes some time :) Let me know if you have any recommendations/experience!
Rob
<img width="1032" alt="Screenshot 2022-03-12 at 09 10 50" src="https://user-images.githubusercontent.com/20374662/158009862-95b453e8-531d-48d2-afd4-7852bb3405ee.png">

**Comment by pharringtonp19** [comment]
@RobertTLange That's a nice figure. I will spend some time playing with the hyperparameters then. Thanks!
